languages/wikipron/src/generate_summary.py

```diff
@@ -21,6 +21,8 @@ def _handle_wiki_name(
     for modifier in modifiers:
         if modifier in language:
             key = file_path[file_path.index("_") + 1:file_path.rindex("_")]
+            if not key:
+                continue
             values = language[modifier][key]
```

Comment:
Did a `KeyError` with `values = languages[modifier][key]` prompt you to add this?  An error like that is something we need to know about so I think logging `file_path` or `language[wiktionary_name]` in `if not key:` would be good.

We can also prepare for Aaron adding another `_` to our TSV names with a slight modification like the following:
```
        if modifier in language:
            k_start = file_path.index("_") + 1
            key = file_path[k_start:file_path.index("_", k_start)]
            ...
```

Reply:
Yeah, what should the log message say?

Reply:
Something like:
```
            logging.info(
                'Failed to isolate key for "%s" modifier in "%s".',
                modifier,
                file_path,
            )
```
Would give us - Failed to isolate key for "script" modifier in "hbs_cyrl_phonemic.tsv".
That ought to be enough information for us to get to the bottom of whatever the problem might be. Did you encounter a `KeyError` after running a scrape of Latin with the dialects? If so that is because you have to get rid of the old Latin files (`lat_phonemic.tsv` and `lat_phonetic.tsv`) before running `generate_summary.py`. Regardless, it's a good idea to catch this error.

Reply:
Done.